bindgen-tests/tests/parse_callbacks/mod.rs

```diff
@@ -146,6 +146,27 @@ impl ParseCallbacks for WrapAsVariadicFn {
     }
 }
 
+#[derive(Debug)]
+struct DeriveTransparentSerialize(String);
```

Comment:
Did you mean to call this `DeriveUppercaseSerialize`? That would make sense to me ;)

Reply:
hahaha yes, I forgot to change that one